src/auth.ts

```diff
@@ -222,7 +221,7 @@ export const loginWithPasskey =
             req.body.clientDataJson,
             [
               ["challenge", challenge],
-              ["origin", secrets.ORIGIN],
+              ["origin", secrets.MIZU_ORIGIN],
```

Comment:
let's not call this MIZU origin, this will serve as the general openid3 backend. We can configure the origin at doppler